timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -238,6 +282,50 @@ public void throwsOnFastForwardWithIncorrectParameter() throws IOException {
         assertThat(response.code()).isEqualTo(HttpStatus.BAD_REQUEST_400);
     }
 
+    private List<Future<LockRefreshToken>> requestLocks(RemoteLockService lockService,
+            int number,
+            ExecutorService executorService,
+            CountDownLatch countDownLatch) {
+        List<Future<LockRefreshToken>> futures = Lists.newArrayList();
+        for (int i = 0; i < number; i++) {
+            int currentTrial = i;
+            futures.add(executorService.submit(() -> {
+                countDownLatch.countDown();
+                countDownLatch.await();
+                return lockService.lock(lockService.toString() + String.valueOf(currentTrial), SLOW_REQUEST);
+            }));
+        }
+        return futures;
+    }
+
+    private int unlockAndCountExceptions(RemoteLockService lockService, List<Future<LockRefreshToken>> futures) {
```

Comment:
Ideally, private methods should appear below where they're called. (Unfortunately, intellij's extract method refactoring often puts methods in the wrong order)